packages/customWidgets/calendar-web/src/components/Calendar.ts

```diff
@@ -139,8 +139,10 @@ class Calendar extends Component<CalendarProps, State> {
     }
 
     private renderCalendar(): ReactNode {
-        const wrapToolbar = (injectedProps: HOCToolbarProps): Function => (toolbarProps: Container.ToolbarProps) =>
-            createElement(CustomToolbar as any, { ...injectedProps, ...toolbarProps });
+        const wrapToolbar =
```

Comment:
is this a result of new version of `prettier`? seems unreasonable file change

Reply:
Yes